rules/opentitan/fpga_cw310.bzl

```diff
@@ -2,14 +2,30 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-load("@lowrisc_opentitan//rules/opentitan:providers.bzl", "Cw310BinaryInfo")
+load(
+    "@lowrisc_opentitan//rules/opentitan:providers.bzl",
+    "Cw305BinaryInfo",
+    "Cw310BinaryInfo",
+    "Cw340BinaryInfo",
```

Comment:
nit: This is in the direction of renaming this file as `fpga_cw` or `fpga_chipwhisperer`.

Reply:
Yep. #19900 suggests doing this rename.